app/src/organisms/OnDeviceDisplay/RobotSettingsDashboard/__tests__/TextSize.test.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react'
+import { fireEvent } from '@testing-library/react'
+
+import { renderWithProviders } from '@opentrons/components'
+
+import { i18n } from '../../../../i18n'
+
+import { TextSize } from '../TextSize'
+
+const mockFunc = jest.fn()
+const render = (props: React.ComponentProps<typeof TextSize>) => {
+  return renderWithProviders(<TextSize {...props} />, {
+    i18nInstance: i18n,
+  })
+}
+
+describe('TextSize', () => {
+  let props: React.ComponentProps<typeof TextSize>
+
+  beforeEach(() => {
+    props = {
+      setCurrentOption: mockFunc,
+    }
+  })
+
+  it('should render text and buttons', () => {
+    const [{ getByText, getByTestId }] = render(props)
+    getByText('Text Size')
+    getByTestId('DisplayTextSize_back_button')
+    getByTestId('DisplayTextSize_decrease')
+    getByTestId('DisplayTextSize_increase')
+  })
+
+  // ToDo (kj:03/03/2023) These cases will be added when text size change method is decided
+  // it('should call mock function when tapping plus button', () => {})
+  // it('should call mock function when tapping minus button', () => {})
```

Comment:
you can leave these as `todo` test cases like:

```suggestion
  it.todo('should call mock function when tapping plus button')
  it.todo('should call mock function when tapping minus button')
```

And then when you run the test, it'll say 2 cases are still `todo`s.